<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sistemadeposicionamiento</class>
 <widget class="QMainWindow" name="Sistemadeposicionamiento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de posiconamiento</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>main</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>150</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>20</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>220</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>setup</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>150</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Descripcion</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Letra</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>331</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Nombre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Letra</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Descripcion</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Letra</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Borrar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_10">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>60</y>
          <width>211</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>391</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_15">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>20</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Calibracion</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_16">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>60</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Calibracion</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_17">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>100</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Horizontal</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_18">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>140</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertical</string>
        </property>
       </widget>
       <widget class="QPushButton" name="up">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>290</y>
          <width>75</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Up</string>
        </property>
        <property name="shortcut">
         <string notr="true"/>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="left">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>320</y>
          <width>75</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Left</string>
        </property>
       </widget>
       <widget class="QPushButton" name="down">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>350</y>
          <width>75</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
       <widget class="QPushButton" name="right">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>320</y>
          <width>75</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Right</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>FAQ´s</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_23">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>511</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Q1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_24">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>511</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Q2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_25">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>511</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Q3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_26">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>511</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Q4</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
